crates/node/p2p/Cargo.toml

```diff
@@ -35,6 +35,7 @@ snap.workspace = true
 futures.workspace = true
 discv5 = { workspace = true, features = ["libp2p"] }
 libp2p = { workspace = true, features = ["macros", "tokio", "tcp", "noise", "gossipsub", "ping", "yamux"] }
+libp2p-identity.workspace = true
```

Comment:
I don't think we need this, the `libp2p` crate itself re-exports the `libp2p-identity` crate through [`libp2p::identity`](https://docs.rs/libp2p/latest/libp2p/identify/index.html)